packages/vue/scripts/sync.sh

```diff
@@ -2,6 +2,9 @@
 
 set -e
 
+# Delete old packages
+rm -rf *.tgz
```

Comment:
Will this work on Windows?

Reply:
Unfortunately no. I'm not familiar with Windows bash commands, do you know if something like this would work?

```
if [[ "$(expr substr $(uname -s) 1 6)" == "CYGWIN" ]]; then
  del /s *.tgz
else
  rm -rf *.tgz
fi
```

Reply:
I've used [rimraf](https://www.npmjs.com/package/rimraf) before and it's worked pretty well. Maybe we could do that and have `rimraf` as a project-level dependency?

Reply:
`rm -rf` works just fine 👀 I just have to do it in a bash terminal (which is already the case with the rest of the script).

Reply:
Perfect! In that case, let's just stick with `rm`.